content/en/docs/concepts/windows/intro.md

```diff
@@ -371,7 +371,7 @@ troubleshooting ideas prior to creating a ticket.
 
 The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control
 plane to manage the cluster it, and nodes to run your workloads.
-[Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
+[Adding Windows nodes](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
 explains how to deploy Windows nodes to your cluster using kubeadm.
```

Comment:
It looks like the /docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ was dropped between v1.24 and v1.25.
The updated link (/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) does not explain how to deploy Windows nodes to the cluster link this paragraph says so we should probably just drop this for now.

There is a WIP doc in kubernetes-sigs/sig-windows-tools for this that is being reviewed https://github.com/kubernetes-sigs/sig-windows-tools/pull/239/files

Reply:
Yes. That page was dropped and now redirected. See: https://github.com/kubernetes/website/blob/main/static/_redirects#L575

Redirections like this pose a challenge for downstream localization teams. If they use the original path, they need to add a new redirect entry per language in the `_redirects` file. We'd better avoid using redirected links in the text.

Reply:
agree with @marosset the redirect doesn't really do the following "explains how to deploy Windows nodes to your cluster using kubeadm."  The steps are essentially the same but things like kubeproxy and cni's require addtional steps on windows.

Reply:
Let's just remove this entire paragraph.

Reply:
Will do.

Reply:
Thanks!